### PR TITLE
[No JIRA] Make iOS project on brand for Backpack

### PR DESCRIPTION
**Pull request by k0nserv**
Just adding an Icon and changing the app name to Backpack

### CODE REVIEW

File: native/ios/native/Images.xcassets/AppIcon.appiconset/Contents.json [hunk 1]
```diff
@@ -31,14 +31,21 @@
...
     },
     {
-      "idiom" : "iphone",
       "size" : "60x60",
+      "idiom" : "iphone",
+      "filename" : "icon@3x.png",
       "scale" : "3x"
+    },
+    {
+      "idiom" : "ios-marketing",
```
Comment (matthewdavidson) [inline native/ios/native/Images.xcassets/AppIcon.appiconset/Contents.json]: What's this? 🤔

File: native/ios/native.xcodeproj/xcshareddata/xcschemes/native.xcscheme [hunk 2]
```diff
@@ -83,6 +84,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = ""
```
Comment (matthewdavidson) [inline native/ios/native.xcodeproj/xcshareddata/xcschemes/native.xcscheme]: This looks superfluous ... 😂

File: native/ios/native/Images.xcassets/AppIcon.appiconset/Contents.json (hunk 1 above)
Comment (k0nserv) [reply native/ios/native/Images.xcassets/AppIcon.appiconset/Contents.json]: Just part of the iconset format

File: native/ios/native.xcodeproj/xcshareddata/xcschemes/native.xcscheme (hunk 2 above)
Comment (k0nserv) [reply native/ios/native.xcodeproj/xcshareddata/xcschemes/native.xcscheme]: These are generated files, who knows :)

**Pull request merged**